data/simpy.gram

```diff
@@ -36,44 +36,50 @@ assert_stmt: 'assert' expression [',' expression]
 
 del_stmt: 'del' targets  # TODO: exclude *target
 
-pass_stmt[stmt_ty]: a='pass' { _Py_Pass(EXTRA(a, a)) }
+pass_stmt[stmt_ty]: a='pass' { _Py_Pass(EXTRA(a, token_type, a, token_type)) }
 
-break_stmt: a='break' { _Py_Break(EXTRA(a, a)) }
+break_stmt: a='break' { _Py_Break(EXTRA(a, token_type, a, token_type)) }
 
-continue_stmt: a='continue' { _Py_Continue(EXTRA(a, a)) }
+continue_stmt: a='continue' { _Py_Continue(EXTRA(a, token_type, a, token_type)) }
 
 import_stmt: import_name | import_from
-import_name[stmt_ty]: a='import' b=dotted_as_names { _Py_Import(b, EXTRA(a, b)) }
+import_name[stmt_ty]: a='import' b=dotted_as_names {
+    _Py_Import(seq_map_to_alias(p, b), EXTRA(a, token_type, seq_get_tail(NULL, b), alias_type)) }
 # note below: the ('.' | '...') is necessary because '...' is tokenized as ELLIPSIS
 import_from[stmt_ty]:
     | a='from' b=('.' | '...')* !'import' c=dotted_name 'import' d=import_from_targets {
-        _Py_ImportFrom(((expr_ty) c)->v.Name.id, d, seq_count_dots(b), EXTRA(a, d)) }
-    | e='from' f=('.' | '...')+ 'import' g=import_from_targets { _Py_ImportFrom(NULL, g, seq_count_dots(f), EXTRA(e, g)) }
+        _Py_ImportFrom(((expr_ty) c)->v.Name.id, seq_map_to_alias(p, d), seq_count_dots(b), EXTRA(a, token_type, seq_get_tail(NULL, d), alias_type)) }
```

Comment:
Do you still need the cast? I'd think not, `dotted_name` is typed correctly. Also a few more times below I believe.

Separately, maybe break up the long line? A line is too long when GitHub starts using a horizontal scroll bar. :-)